app/aws/xray_celery_handlers.py

```diff
@@ -20,7 +20,7 @@
 def xray_before_task_publish(
     sender=None, headers=None, exchange=None, routing_key=None, properties=None, declare=None, retry_policy=None, **kwargs
 ):
-    logger.info(f"xray-celery: before publish: sender={sender}, headers={headers}, kwargs={kwargs}")
+    logger.debug(f"xray-celery: before publish: sender={sender}, headers={headers}, kwargs={kwargs}")
```

Comment:
Slight change unrelated to the scope of this PR to reduce the log noise around x-ray in production and staging. This was previously useful to debug x-ray traces when it wasn't working properly.